<launch>
    <arg name="config_file" default="$(find minau_tools)/config/repeat.csv"/>
    <arg name="data_location" default="$(find minau_tools)/repeat_data"/>
    <node name="repeat_tester" pkg="minau_tools" type="repeat_tester.py" output="screen">
        <param name="file" value="$(arg config_file)"/>
        <param name="data_loc" value="$(arg data_location)"/>
    </node>
</launch>
